docs/getting_started/index.md

```diff
@@ -116,62 +116,112 @@ While we've used Taipy's Python APIs to handle our scenario, it usually works se
+            state.scenario.input_name.write(state.input_name)
+            state.scenario.submit(wait=True)
+            state.message = scenario.message.read()
 
-Now, let’s explain the key elements of this code:
 
-```python
-import taipy as tp
+        if __name__ == "__main__":
+            tp.Core().run()
+            scenario = tp.create_scenario(scenario_cfg)
+            tp.Gui(page).run()
+        ```
 
+        [Get the whole code](./hello_world_md.py){: .tp-btn target='blank'}
+
+        Now, let’s explain the key elements of this code:
+
+        ```python
+        import taipy as tp
+
+
+        page = """
+        Name: <|{input_name}|input|>
+        <|submit|button|on_action=submit_scenario|>
+        Message: <|{message}|text|>
+        """
+        ```
+
+    === "Python"
+
+        Taipy pages can be defined in multiple ways: Markdown, Html or Python. *page* is built through the Taipy Gui Builder.
+
+        ```python linenums="1"
+        import taipy as tp
+        import taipy.gui.builder as tgb
+
+        # Previous configuration of scenario
+        ...
+
+        with tgb.Page() as page:
+            tgb.text("Name:")
+            tgb.input("{input_name}")
+            tgb.button("Submit", on_action=submit_scenario)
+            tgb.text("Message {message}")
+
+
+        input_name = "Taipy"
+        message = None
+
+
+        def submit_scenario(state):
+            state.scenario.input_name.write(state.input_name)
+            state.scenario.submit(wait=True)
+            state.message = scenario.message.read()
+
+
+        if __name__ == "__main__":
+            tp.Core().run()
+            scenario = tp.create_scenario(scenario_cfg)
+            tp.Gui(page).run()
+        ```
+
+        [Get the whole code](./hello_world_tgb.py){: .tp-btn target='blank'}
+
+        Now, let’s explain the key elements of this code:
+
+        ```python
+        import taipy as tp
 
-page = """
-Name: <|{input_name}|input|>
-<|submit|button|on_action=submit_scenario|>
-Message: <|{message}|text|>
-"""
-```
 
-**Pages**
+        with tgb.Page() as page:
+            tgb.text("Name:")
+            tgb.input("{input_name}")
+            tgb.button("Submit", on_action=submit_scenario)
+            tgb.text("Message {message}")
+        ```
 
-Taipy pages can be defined in multiple ways: Markdown, Html or Python. The *page* 
-variable is a Markdown representation of the user interface. 
-It uses standard Markdown syntax as well as visual elements.
 
 **Visual elements**
 
 Taipy offers various visual elements that can interact with your Python variables and 
 environment. It allows you to display variables, modify them, and interact with the application.
 
-Here is a generic example of one: `<|{variable}|visual_element_type|...|>`. *variable* is 
+Here is a generic example of one: `<|{variable}|visual_element_type|...|>` with Markdown or `tgb.visual_element_type("{variable}", ...)` with the Python API. *variable* is 
```

Comment:
Same remark here (Simplified tgb syntax)
-> `tgb.visual_element_type(variable, ...)`

"with the Python API" -> "with the Page Builder API"

Reply:
Please break line before 100 chars.